kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -19,14 +19,18 @@ package org.apache.kyuubi.engine
 
 import java.io.File
 import java.net.{URI, URISyntaxException}
+import java.nio.file.Path
 import java.util.{Locale, ServiceLoader}
 
 import scala.collection.JavaConverters._
 import scala.util.control.NonFatal
 
+import org.apache.hadoop.shaded.org.apache.commons.io.FileUtils
```

Comment:
```
import org.apache.commons.io.FileUtils
```

Reply:
`commons-io` is not available in the server module, shall we add this to dependency?

Reply:
If that, you can use Utils.deleteDirectoryRecursively instead.

Reply:
Added `commons-io` dependency to server module, and use it in FileUtils replacing `import org.apache.hadoop.shaded.org.apache.commons.io.FileUtils`.
.

Reply:
Addressed.